pkg/network/opencontrail/controller_test.go

```diff
@@ -2112,3 +2112,151 @@ func TestServiceBeforeNamespace(t *testing.T) {
 	_, err := types.VirtualNetworkByName(client, "default-domain:newns:service-foo")
 	assert.NoError(t, err)
 }
+
+
+func TestDomainVariable(t *testing.T) {
+	kube := mocks.NewKubeClient()
+
+	client := new(contrail_mocks.ApiClient)
+	client.Init()
+
+	client.AddInterceptor("virtual-machine-interface", &VmiInterceptor{})
+	client.AddInterceptor("virtual-network", &NetworkInterceptor{})
+
+	allocator := new(mocks.AddressAllocator)
+
+	controller := NewTestController(kube, client, allocator, nil)
```

Comment:
pass a nil allocator unless you want to override the allocator for some reasons.